test/test_ff_utils.py

```diff
@@ -603,6 +603,146 @@ def test_expand_es_metadata(integrated_ff):
+    status.update(for_all)
+    resp = ff_utils.search_facets(base_url=url, ff_env=ff_env, key=key, kwargs=status)
+    assert len(resp) == 12
+    warnings = {'Warnings': 'No value'}
+    warnings.update(for_all)
+    resp = ff_utils.search_facets(kwargs=warnings)
+    assert len(resp) == 4
+    both_projects = {'Project': '4DN|External'}
+    both_projects.update(for_all)
+    resp = ff_utils.search_facets(kwargs=both_projects)
+    assert len(resp) == 13
+    both_labs = {'Lab': '4DN Testing Lab|Some Other Guys lab'}
+    both_labs.update(for_all)
+    resp = ff_utils.search_facets(kwargs=both_labs)
+    assert len(resp) == 13
+    proj_exp_type = {'Project': '4DN', 'Experiment Type': 'Dilution Hi-C'}
+    proj_exp_type.update(for_all)
+    resp = ff_utils.search_facets(kwargs=proj_exp_type)
+    assert len(resp) == 2
+    proj_exp_type = {'Project': '4DN|External', 'Experiment Type': 'Dilution Hi-C|2-stage Repli-seq'}
+    proj_exp_type.update(for_all)
+    resp = ff_utils.search_facets(kwargs=proj_exp_type)
+    assert len(resp) == 5
+    proj_exp_sam = {'Project': '4DN|External',
+                    'Experiment Type': 'Dilution Hi-C|2-stage Repli-seq',
+                    'Sample Type': 'in vitro differentiated cells'}
+    proj_exp_sam.update(for_all)
+    resp = ff_utils.search_facets(kwargs=proj_exp_sam)
+    assert len(resp) == 1
+    exp_sam = {'Experiment Type': 'ATAC-seq', 'Sample': 'GM12878'}
+    exp_sam.update(for_all)
+    resp = ff_utils.search_facets(kwargs=exp_sam)
+    assert len(resp) == 1
+    exp_sam_data = {'Experiment Category': 'Sequencing', 'Sample': 'GM12878',
+                    'Dataset': 'Z et al. 2-Stage Repliseq'}
+    exp_sam_data.update(for_all)
+    resp = ff_utils.search_facets(kwargs=exp_sam_data)
+    assert len(resp) == 2
+
+
+@pytest.mark.integrated
+def test_search_facets_users(integrated_ff):
+    """ Tests search_facets with users intead of experiment set """
+    key, ff_env = integrated_ff['ff_key'], integrated_ff['ff_env']
+    url = integrated_ff['ff_key']['server']
+    all_facets = ff_utils.get_item_facets('user', key=key, ff_env=ff_env)
+    affiliation = {'item_type': 'user',
+                   'Affiliation': '4DN Testing Lab',
+                   'key': key,
+                   'ff_env': ff_env,
+                   'base_url': url,
+                   'item_facets': all_facets}
+    resp = ff_utils.search_facets(kwargs=affiliation)
+    assert len(resp) == 4
+
+
+@pytest.mark.integrated
+def test_get_qc_metrics(integrated_ff):
+    """
+    Tests that we correctly extract qc metric uuids (and therefore items) from the helper
+    """
+    key, ff_env = integrated_ff['ff_key'], integrated_ff['ff_env']
+    uuid = '331106bc-8535-3338-903e-854af460b544'
+    qc_metrics = ff_utils.get_associated_qc_metrics(uuid, key=key, ff_env=ff_env)
+    assert '4c9dabc6-61d6-4054-a951-c4fdd0023800' in qc_metrics
+    assert '131106bc-8535-4448-903e-854abbbbbbbb' in qc_metrics
```

Comment:
Maybe check the values as well as kets of `qc_metrics` to confirm that they are indeed QualityMetric items (e.g. check `@type` in the objects)